### PR TITLE
Clean platform_map in generateDisabledTestListJson

### PR DESCRIPTION
**Pull request by tejaswini0604**
Made code changes as per Clean platform_map in generateDisabledTestListJson issue

### CODE REVIEW

File: disabledTestParser/generateDisabledTestListJson.py [hunk 1]
```diff
@@ -5,16 +5,14 @@
 import re
 
 platform_map = {
-    "arm-linux": ["arm_linux"],
     "linux-aarch64": ["aarch64_linux"],
-    "linux-arm": ["arm_linux"],
```
Comment (llxia) [inline disabledTestParser/generateDisabledTestListJson.py:10]: Line 10 is valid. We should keep it.

Comment (tejaswini0604) [reply disabledTestParser/generateDisabledTestListJson.py:10]: Added this line

**Review by llxia — APPROVED** [review]
Thanks @tejaswini0604

**Review by smlambert — APPROVED** [review]
thanks @tejaswini0604 !

**Pull request merged**